hack/install.sh

```diff
@@ -237,7 +266,7 @@ do_install() {
 			exit 0
 			;;
 
-		fedora|centos)
+		fedora|centos|oracleserver)
```

Comment:
Except oracleserver is Oracle Linux's lsb_dist value. That's what is returned by lsb_release and what was assigned to us by LSB.

If you must have it as oraclelinux, then you'll need to split out oracleserver as the lsb_dist and replace lsb_dist with oraclelinux.